src/data/thunks.js

```diff
@@ -150,10 +150,22 @@ export function startProctoredExam() {
         .then(() => updateAttemptAfter(
           exam.course_id, exam.content_id, continueAttempt(attempt.attempt_id),
         )(dispatch))
-        .catch(() => handleAPIError(
-          { message: 'Something has gone wrong starting your exam. Please double-check that the application is running.' },
-          dispatch,
-        ));
+        .catch(error => {
+          if (error) {
+            logInfo(
```

Comment:
Why logInfo instead of just logError here?

Reply:
I thought about it initially, but I'm afraid it'll blow up the error rate, especially if there is user error involved.

Reply:
It's a fair concern. I still think we should do logError. I thought we can just exclude this one on newrelic? Or at least set the alert threshold high?
To me, this is a good error to alert us anyways to actively help and identify learner issues